pcdsdevices/epics_motor.py

```diff
@@ -125,7 +127,38 @@ def limits(self, limits):
         self.low_limit = limits[0]
         self.high_limit = limits[1]
 
+    def enable(self, enable):
+        """
+        Sets the state for the motor
+
+        Parameters
+        ----------
+        enable: Bool
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.disabled.set(not enable)
+
     def check_value(self, value):
+        """
+        Check if the motor is disabled
+
+        Raises
+        ------
+        Exception
+        """
+        # First check that the user has returned a valid EPICS value.
+        super().check_value(value)
+
+        # Find the value for the disabled attribute
+        is_disabled = self.disabled.value
+        if is_disabled == 1:
+            raise Exception("Motor is not enabled. Motion requests "
+                            "ignored")
+
```

Comment:
Almost! You made a function with two docstrings that calls `super().check_value` twice. Make this a single function with a single docstring. 

Also condense the if statement by just comparing against value specifically. No need to make the `is_disabled` variable.

Reply:
@teddyrendahl I merged the doctrings - I wasn't sure if I just keep it in order of what was called, or if I keep the Raises sections together and the descriptions together